File: datasets/IR-Plag/convertai-original-java-rest-python/case-06/plagiarized/L6/08/Kasus6L6.py
import sys

def main():
    # Delarasi
    arr_num = [0] * 10
    inp = sys.stdin

    # for untuk 10 kali input
    for i in range(9, -1, -1):
        # Read a number
        print("Read a number: ", end="")
        arr_num[i] = int(inp.readline().strip())

    # Display the array
    reverse(arr_num)

def reverse(arr):
    for num in arr:
        print(num)

if __name__ == "__main__":
    main()

File: datasets/IR-Plag/intellij-original-java-rest-kotlin/case-06/plagiarized/L6/08/Kasus6L6.kt
package IR

import java.util.*

/*
* To change this license header, choose License Headers in Project Properties.
* To change this template file, choose Tools | Templates
* and open the template in the editor.
*/

/**
 *
 * @author B15130F5DDB6B5F1622EF91DAC4C1AAE
 */
object Kasus6L6 {
    @JvmStatic
    fun main(args: Array<String>) {
        //Delarasi
        val arrNum = IntArray(10)
        val inp = Scanner(System.`in`)
        // for untuk 10 kali input
        var i = 9
        while (i >= 0) {
            // Read a number
            print("Read a number: ")
            arrNum[i] = inp.nextInt()
            i--
        }


        // Display the array
        Reverse(arrNum)
    }

    private fun Reverse(arr: IntArray) {
        var i = 0
        while (i < 10) {
            //Print angka dalam array
            println(arr[i])
            i++
        }
    }
}
